freqtrade/strategy/backtest_lookahead_bias_checker.py

```diff
@@ -0,0 +1,226 @@
+import copy
+from copy import deepcopy
+from datetime import datetime, timedelta, timezone
+
+import pandas
+
+from freqtrade.configuration import TimeRange
+from freqtrade.data.history import get_timerange
+from freqtrade.exchange import timeframe_to_minutes
+from freqtrade.optimize.backtesting import Backtesting
+
+
+class BacktestLookaheadBiasChecker:
+    class VarHolder:
+        timerange: TimeRange
+        data: pandas.DataFrame
+        indicators: pandas.DataFrame
+        result: pandas.DataFrame
+        compared: pandas.DataFrame
+        from_dt: datetime
+        to_dt: datetime
+        compared_dt: datetime
+
+    class Analysis:
+        def __init__(self):
+            self.total_signals = 0
+            self.false_entry_signals = 0
+            self.false_exit_signals = 0
+            self.false_indicators = []
+            self.has_bias = False
+
+        total_signals: int
+        false_entry_signals: int
```

Comment:
I really don't like this being a nested class (nor the above)
there's little reason for that - and it makes usage kinda odd (same for the class above, obviously).

Reply:
done